src/main/java/com/cleanroommc/groovyscript/compat/mods/prodigytech/ZorraAltar.java

```diff
@@ -0,0 +1,75 @@
+package com.cleanroommc.groovyscript.compat.mods.prodigytech;
+import com.cleanroommc.groovyscript.api.GroovyBlacklist;

```

Comment:
there should be a linebreak between package and the import block.

Reply:
I think when I was writing that file Pycharm crashed and removed all newlines from the file. Will fix.